tests/controllers/sellerReview.test.ts

```diff
@@ -0,0 +1,156 @@
+    expect(getAllSellerReviewsQuery.data?.length).toBe(2);
+    expect(getAllSellerReviewsQuery.data?.at(1)?._id).toEqual(
+      mockSellerReview2._id
+    );
+  });
+
+  it('Should be able to fetch SellerReview by ID', async () => {
+    let getSellerReviewByIdQuery = await getSellerReviewById(
+      mockSellerReview._id.toString()
+    );
+    expect(getSellerReviewByIdQuery.status).toBe(true);
+    expect(getSellerReviewByIdQuery.data).toBeInstanceOf(SellerReview);
+    expect(getSellerReviewByIdQuery.data?._id).toEqual(mockSellerReview._id);
+    expect(getSellerReviewByIdQuery.data?.request._id).toEqual(
+      mockBuyerRequest._id
+    );
+    expect(getSellerReviewByIdQuery.data?.service._id).toEqual(
+      mockSellerService._id
+    );
+    expect(getSellerReviewByIdQuery.data?.seller._id).toEqual(
+      mockSellerUser._id
+    );
+    expect(getSellerReviewByIdQuery.data?.buyer._id).toEqual(mockBuyerUser._id);
+    expect(getSellerReviewByIdQuery.data?.comment).toEqual(
+      mockSellerReview.comment
+    );
+    expect(getSellerReviewByIdQuery.data?.rating).toEqual(
+      mockSellerReview.rating
+    );
+  });
+
+  it('Should be able to fetch SellerReviews by Seller User', async () => {
+    let getSellerReviewsBySellerUserQuery = await getSellerReviewsBySellerUser(
+      mockSellerUser._id.toString()
+    );
+    expect(getSellerReviewsBySellerUserQuery.status).toBe(true);
+    expect(getSellerReviewsBySellerUserQuery.data).toBeInstanceOf(Array);
+    expect(getSellerReviewsBySellerUserQuery.data?.length).toBe(2);
+
+    getSellerReviewsBySellerUserQuery = await getSellerReviewsBySellerUser(
+      mockBuyerUser._id.toString()
+    );
+    expect(getSellerReviewsBySellerUserQuery.status).toBe(true);
+    expect(getSellerReviewsBySellerUserQuery.data).toBeInstanceOf(Array);
+    expect(getSellerReviewsBySellerUserQuery.data?.length).toBe(0);
+  });
+
+  it('Should be able to fetch SellerReviews by Buyer User', async () => {
+    let getSellerReviewsByPostingUserQuery =
+      await getSellerReviewsByPostingUser(mockBuyerUser._id.toString());
+    expect(getSellerReviewsByPostingUserQuery.status).toBe(true);
+    expect(getSellerReviewsByPostingUserQuery.data).toBeInstanceOf(Array);
+    expect(getSellerReviewsByPostingUserQuery.data?.length).toBe(2);
+
+    getSellerReviewsByPostingUserQuery = await getSellerReviewsByPostingUser(
+      mockSellerUser._id.toString()
+    );
+    expect(getSellerReviewsByPostingUserQuery.status).toBe(true);
+    expect(getSellerReviewsByPostingUserQuery.data).toBeInstanceOf(Array);
+    expect(getSellerReviewsByPostingUserQuery.data?.length).toBe(0);
+  });
+
+  it('Should be able to create a SellerReview', async () => {
+    let createSellerReviewQuery = await createSellerReview(
+      mockBuyerRequest._id.toString(),
+      'Seller Review created inside the test',
+      2.3
+    );
+
+    expect(createSellerReviewQuery.status).toBe(true);
```

Comment:
Instead of checking what's been returned from `createSellerReviewQuery`, fetch the SellerReview by ID and check if it matches since the API might return the SellerReviw but we need to make sure it's been inserted in the database.

Reply:
Done